### PR TITLE
Why is my "Save" button disabled? FAQ section

### PR DESCRIPTION
**Pull request by annasun19**
### What does this PR do?
Create a new FAQ section explaining why some Metrics without Limits tag configurations cause the save button to be disabled.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by apigirl** [comment]
the editorial review tracked in DOCS-4012

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Why is my "Save" button disabled?
+kind: faq
+is_beta: false
+---
+This is not a bug but intentional. The "Save" button is disabled in infrequent scenarios where it is more cost effective to leave this metric entirely unconfigured without using Metrics without Limits
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:6]:
```suggestion
It is possible for a proposed configuration to result in an indexed custom metrics volume that is larger than the original ingested volume. The "Save" button is intentionally disabled in infrequent scenarios where it is more cost effective to leave a metric entirely unconfigured without using Metrics without Limits™. 
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Why is my "Save" button disabled?
+kind: faq
+is_beta: false
+---
+This is not a bug but intentional. The "Save" button is disabled in infrequent scenarios where it is more cost effective to leave this metric entirely unconfigured without using Metrics without Limits 
+
+As a quick refresher on Metrics without Limits (more details [here](https://docs.datadoghq.com/metrics/metrics-without-limits/)), you can continue sending us their originally submitted volume of metrics with no code level changes and define a tag configuration to keep only a smaller subset of custom metrics to index.
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:8]:
```suggestion
Using [Metrics without Limits™][1], you can continue sending Datadog your originally submitted volume of metrics with no code-level changes, and define a tag configuration to keep only a smaller subset of custom metrics to index. 
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: Why is my "Save" button disabled?
+kind: faq
+is_beta: false
+---
+This is not a bug but intentional. The "Save" button is disabled in infrequent scenarios where it is more cost effective to leave this metric entirely unconfigured without using Metrics without Limits 
+
+As a quick refresher on Metrics without Limits (more details [here](https://docs.datadoghq.com/metrics/metrics-without-limits/)), you can continue sending us their originally submitted volume of metrics with no code level changes and define a tag configuration to keep only a smaller subset of custom metrics to index. 
+
+By using MWL, your original raw metric’s data must now be recombined/reaggregated and stored against that smaller volume of indexed custom metrics in order to preserve mathematically accurate query results. Therefore for each of the remaining indexed custom metrics, we store your specified number of time/space aggregations.
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:10]:
```suggestion
By using Metrics without Limits™, your original raw metrics data must be recombined and re-aggregated, and stored against that smaller volume of indexed custom metrics in order to preserve mathematically accurate query results. Therefore, for each of the remaining indexed custom metrics, Datadog stores your specified number of time/space aggregations. 
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+kind: faq
+is_beta: false
+---
+This is not a bug but intentional. The "Save" button is disabled in infrequent scenarios where it is more cost effective to leave this metric entirely unconfigured without using Metrics without Limits 
+
+As a quick refresher on Metrics without Limits (more details [here](https://docs.datadoghq.com/metrics/metrics-without-limits/)), you can continue sending us their originally submitted volume of metrics with no code level changes and define a tag configuration to keep only a smaller subset of custom metrics to index. 
+
+By using MWL, your original raw metric’s data must now be recombined/reaggregated and stored against that smaller volume of indexed custom metrics in order to preserve mathematically accurate query results. Therefore for each of the remaining indexed custom metrics, we store your specified number of time/space aggregations. 
+
+The resulting number of indexed custom metrics for your MWL configuration is (`the # of remaining tag value combinations`–specified by your tag configuration) x (`the # of time/space aggregations`–specified by the Customize Aggregations section)
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:12]:
```suggestion
The resulting number of indexed custom metrics for your Metrics without Limits™ configuration is (`the number of remaining tag value combinations`—specified by your tag configuration) x (`the number of time/space aggregations`—specified by the Customize Aggregations section)
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+**Example**
+Suppose we want to use Metrics without Limits to reduce the cardinality of the `shopist.basket.size` metric.
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:15]:
```suggestion
Suppose you want to use Metrics without Limits™ to reduce the cardinality of the `shopist.basket.size` metric. 
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+By using MWL, your original raw metric’s data must now be recombined/reaggregated and stored against that smaller volume of indexed custom metrics in order to preserve mathematically accurate query results. Therefore for each of the remaining indexed custom metrics, we store your specified number of time/space aggregations. 
+
+The resulting number of indexed custom metrics for your MWL configuration is (`the # of remaining tag value combinations`–specified by your tag configuration) x (`the # of time/space aggregations`–specified by the Customize Aggregations section)
+
+**Example**
+Suppose we want to use Metrics without Limits to reduce the cardinality of the `shopist.basket.size` metric. 
+
+{{< img src="metrics/faq/all-tags.jpg" alt="All Tags Configuration">}}
+
+Let’s assume `shopist.basket.size` reports values against 4 tag value combinations, like {host: a, region: us, env: prod}–that is, `shopist.basket.size` originally emits **4 custom metrics** as shown in the diagram below.
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:19]:
```suggestion
Assuming `shopist.basket.size` reports values against four tag value combinations, such as {host: a, region: us, env: prod}—that is, `shopist.basket.size` originally emits **four custom metrics** as shown in the following diagram:
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+{{< img src="metrics/faq/all-tags-diagram.jpg" alt="Custom Metric Breakdown of All Tags Configuration">}}
+
+If we use MWL to tag configure only the {region, env} tags, only three tag value combinations remain:
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:23]:
```suggestion
Using Metrics without Limits™ to tag configure only the `{region, env}` tags, only three tag value combinations remain:
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+1. {region:us, env:prod}
+2. {region:eu, env:prod}
+3. {region:us, env:staging}
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:26]:
```suggestion
* `{region:us, env:prod}`
* `{region:eu, env:prod}`
* `{region:us, env:staging}`
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+{{< img src="metrics/faq/disabled-save.png" alt="Disabled Save Configuration">}}
+
+The resulting number of custom metrics from this proposed configuration is as follows: (`the # of remaining tag value combinations`) x (`the # of time/space aggregations`) = (3) x (2) = **6 custom metrics**.
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:30]:
```suggestion
The resulting number of custom metrics from this proposed configuration is as follows: (`the number of remaining tag value combinations`) x (`the number of time/space aggregations`) = `(3) x (2)` = **6 custom metrics**.
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+{{< img src="metrics/faq/mwl-diagram.jpg" alt="Custom Metric Breakdown of MWL Configuration">}}
+
+Consequently, it’s possible for a proposed configuration to result in an indexed custom metrics volume (6 custom metrics) that is larger than the original ingested volume (4 custom metrics). **In these cases, it is better to leave this metric unconfigured on the “All Tags” toggle option for the most cost optimized configuration of this metric.**
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:34]:
```suggestion
Therefore, it is possible for a proposed configuration to result in an indexed custom metrics volume (6 custom metrics) that is larger than the original ingested volume (4 custom metrics). **In these cases, it is better to leave this metric unconfigured with the “All Tags” toggle option for the most cost-optimized configuration of this metric.**

[1]: /metrics/metrics-without-limits/
```

File: content/en/metrics/faq/why-is-my-save-button-disabled.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+---
+title: Why is my "Save" button disabled?
```
Comment (jtappa) [inline content/en/metrics/faq/why-is-my-save-button-disabled.md:2]:
```suggestion
title: Why is my "Save" button disabled when configuring metrics tags?
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**